db/CouriersDB.go

```diff
@@ -0,0 +1,62 @@
+package db
+
+import (
+	"fmt"
+	"strconv"
+)
+
+type Courier struct {
+	IdCourier        uint16 `json:"id_courier"`
+	CourierName      string `json:"courier_name"`
+	ReadyToGo        bool   `json:"ready_to_go"`
+	PhoneNumber      string `json:"phone_number"`
+	Email            string `json:"email"`
+	Rating           uint16 `json:"rating"`
+	Photo            string `json:"photo"`
+	NumberOfFailures uint16 `json:"number_of_failures"`
```

Comment:
I don't see anything about rating and number of failures in ticket description. I even doubt that they would be stored this way like a regular fields in db, they are often computed with some extra rules and if needed cached (e.g. do you really need to know all failures even when they failed delivery 2 years ago?)